x/emissions/keeper/keeper.go

```diff
@@ -210,13 +213,20 @@ func NewKeeper(
 	ak AccountKeeper,
 	bk BankKeeper,
 	feeCollectorName string,
+	config Config,
 ) Keeper {
+	defaultConfig := DefaultConfig()
+	if config.MaxMetadataLen == 0 {
+		config.MaxMetadataLen = defaultConfig.MaxMetadataLen
+	}
+
 	sb := collections.NewSchemaBuilder(storeService)
 	k := Keeper{
 		cdc:                                      cdc,
 		storeService:                             storeService,
 		addressCodec:                             addressCodec,
 		feeCollectorName:                         feeCollectorName,
+		Config:                                   config,
```

Comment:
@guilherme-brandao is this really the best way to do this, seems like a big change to store some new config object on the keeper, we should have to update InitGenesis as well, etc

Reply:
I think moduleParams is less awkward way to go...

i had thought we were sourcing something from cosmos-sdk

Reply:
fixed in #588